items/items_init.py

```diff
@@ -0,0 +1,28 @@
+from items.weapon import Weapon
+from items.armour import Armour
+from items.potion import Potion
+
+
+class InitItems:
+
+    @staticmethod
+    def init_weapon(name: str, config: dict) -> Weapon:
+        weapon_name = config.get(name).get("weapon").get("name")
+        power = config.get(name).get("weapon").get("power")
+        return Weapon(weapon_name, power)
+
+    @staticmethod
+    def init_armour(name: str, config: dict) -> list[Armour]:
+        equipment = []
+        for part_arm in config.get(name).get("armour"):
+            part = part_arm.get("part")
+            protect = part_arm.get("protection")
+            equipment.append(Armour(part, protect))
+        return equipment

```

Comment:
You can use unpacking `**` and list comprehension here. You can even skip using `get` to access config values. You anyway don't want them to be None, so you can skip it here